activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -187,6 +215,10 @@ def target_version
         ENV["VERSION"].to_i if ENV["VERSION"] && !ENV["VERSION"].empty?
       end
 
+      def dry_run
+        ENV["DRY_RUN"] ? ENV["DRY_RUN"] == "true" : false
```

Comment:
What do you think on just checking that something is that that environment variable is not `nil`? I think that some developers may think that `DRY_RUN=1 [...]` would work

Something like:
```ruby
def dry_run
  !!ENV["DRY_RUN"]
end
```